Content.Server/Radio/EntitySystems/JammerSystem.cs

```diff
@@ -21,32 +25,57 @@ public override void Initialize()
         SubscribeLocalEvent<ActiveRadioJammerComponent, PowerCellChangedEvent>(OnPowerCellChanged);
         SubscribeLocalEvent<RadioJammerComponent, ExaminedEvent>(OnExamine);
         SubscribeLocalEvent<RadioSendAttemptEvent>(OnRadioSendAttempt);
+        SubscribeLocalEvent<RadioJammerComponent, GetVerbsEvent<Verb>>(OnGetVerb);
     }
 
     public override void Update(float frameTime)
     {
         var query = EntityQueryEnumerator<ActiveRadioJammerComponent, RadioJammerComponent>();
+
         while (query.MoveNext(out var uid, out var _, out var jam))
         {
-            if (_powerCell.TryGetBatteryFromSlot(uid, out var battery) &&
-                !battery.TryUseCharge(jam.Wattage * frameTime))
+            if (_powerCell.TryGetBatteryFromSlot(uid, out var battery))
             {
-                RemComp<ActiveRadioJammerComponent>(uid);
+                if (!battery.TryUseCharge(GetCurrentWattage(jam) * frameTime))
+                {
```

Comment:
Doesn't PowerCellDraw handle this now?

Also if you're doing RemComp on something being iterated it should really be RemCompDeferred

Reply:
Hey I'm having some issues with the recent commit. It seems to crash instantly whenever I try to use the jammer. To be clear: this is master WITHOUT any of my changes committed (At least I think). Here is a video:

[jammersad.webm](https://github.com/space-wizards/space-station-14/assets/107373427/e3409e82-2fb0-4121-85ba-9fae7bb85b22)

Could you check if it works on your end or if its just me?
(The PR that I think is breaking stuff)
https://github.com/space-wizards/space-station-14/pull/25871

Reply:
I can confirm this error. That's my fault - insufficient testing on my part. I'll PR a fix ASAP.

Reply:
Fix: #25880

Reply:
Oh thanks a lot! I got confused because there is also another problem that is for sure my fault (Or gits) :sob: